src/core/server/http/router/route.ts

```diff
@@ -42,13 +54,64 @@ export interface RouteConfigOptions {
    * Additional metadata tag strings to attach to the route.
    */
   tags?: readonly string[];
+
+  /**
+   * Default value: allows parsing of the following mime types:
+   * * application/json
+   * * application/*+json
+   * * application/octet-stream
+   * * application/x-www-form-urlencoded
+   * * multipart/form-data
+   * * text/*
+   * A string or an array of strings with the allowed mime types for the endpoint. Use this settings to limit the set of allowed mime types. Note that allowing additional mime types not listed
+   * above will not enable them to be parsed, and if parse is true, the request will result in an error response.
+   */
+  accepts?: Method extends 'get'
+    ? never
+    : RouteContentType | RouteContentType[] | string | string[];
+
+  /**
+   * Default value: The one set in the kibana.yml config file under the parameter `server.maxPayloadBytes`.
+   * Limits the size of incoming payloads to the specified byte count. Allowing very large payloads may cause the server to run out of memory.
+   */
+  maxBytes?: Method extends 'get' ? never : number;
```

Comment:
NIT: I would have put the default value explanation after the actual description of the fields.

Reply:
I agree! The pre-existing params provide the default value at the end. Updated.